### PR TITLE
Update reprensentating -qubit-state.ipynb(easy qubit state representation)

### PR DESCRIPTION
**Pull request by rajat709**
here i have added line [3275-3278]
for visual representation of qubit it's give clear state of qubit and show how different gate acts on qubit state,
and also difines all gates in easy way!

# Changes made

# Justification

### CODE REVIEW

File: content/ch-states/representing-qubit-states.ipynb [hunk 1]
```diff
@@ -3271,7 +3271,11 @@
     "\n",
     "Below we have plotted a qubit in the state $|{+}\\rangle$. In this case, $\\theta = \\pi/2$ and $\\phi = 0$.\n",
     "\n",
-    "(Qiskit has a function to plot a bloch sphere, `plot_bloch_vector()`, but at the time of writing it only takes cartesian coordinates. We have included a function that does the conversion automatically).\n"
+    "(Qiskit has a function to plot a bloch sphere, `plot_bloch_vector()`, but at the time of writing it only takes cartesian coordinates. We have included a function that does the conversion automatically).\n",
+    "\n",
+    "(or)\n",
+    "\n",
+    "(For custom visual representation of qubit state [click here](https://javafxpert.github.io/grok-bloch/).\n"
```
Comment (frankharkins) [inline content/ch-states/representing-qubit-states.ipynb:3278]:
```suggestion
    "You can also try [this interactive Bloch sphere demo](https://javafxpert.github.io/grok-bloch/).\n"
```

File: content/ch-states/representing-qubit-states.ipynb [hunk 2]
```diff
@@ -3271,7 +3271,11 @@
     "\n",
     "Below we have plotted a qubit in the state $|{+}\\rangle$. In this case, $\\theta = \\pi/2$ and $\\phi = 0$.\n",
     "\n",
-    "(Qiskit has a function to plot a bloch sphere, `plot_bloch_vector()`, but at the time of writing it only takes cartesian coordinates. We have included a function that does the conversion automatically).\n"
+    "(Qiskit has a function to plot a bloch sphere, `plot_bloch_vector()`, but at the time of writing it only takes cartesian coordinates. We have included a function that does the conversion automatically).\n",
+    "\n",
+    "(or)\n",
```
Comment (frankharkins) [inline content/ch-states/representing-qubit-states.ipynb:3276]:
```suggestion
```

**Review by frankharkins — CHANGES_REQUESTED** [review]
Happy to link to this demo, but think the wording needs changing a bit.

**Comment by rajat709** [comment]
Sure sir, you rectify it!

**Review by frankharkins — APPROVED** [review]
Thanks!

**Pull request merged**